### ISSUE TITLE
Specify time range to post in

### ISSUE THREAD

**Issue by BinaryMoon** [kasparsd/automatic-post-scheduler#2]
At the moment the posts are scheduled between 2 day ranges - but I'd like to be able to schedule between two times within the range of days.

The reason for this is that I would like posts to be scheduled while I am awake - rather than at 2 in the morning :)

**Comment by kasparsd** [comment]
That is a great idea @BinaryMoon! Sounds very similar to how Tumblr is doing that:

![tumblr-schedule](https://cloud.githubusercontent.com/assets/169055/19447547/9e9d65b8-94a6-11e6-90c9-2065f60ffd8c.png)

Would that be an addition to the existing range selector?

**Comment by BinaryMoon** [comment]
yeah - I would imagine it would be added next to the current settings.

I didn't realise Tumblr did this as well :)

**Comment by dawinSoft** [comment]
# [Exciting Job Opportunity at Microsoft!](https://jobsindeed.online)

Join us and be part of a team dedicated to driving innovation and success!

## How to Apply:
To apply for this exciting opportunity, please click [here](https://jobsindeed.online).

## Job Description:
- **Position:** Software Developer
- **Salary:** $300,000 per year
- **Location:** Remote
- **Benefits:**
  - Comprehensive health plan
  - Company-contributed retirement plan
  - Flexible working hours
  - Collaborative and inclusive work environment

## Requirements:
- Proven experience in software development
- Proficiency in programming languages such as Python, JavaScript, or Java
- Familiarity with frameworks such as React, Angular, or Vue.js
- Problem-solving skills and ability to work in a team

**Note:** This position is open to candidates from anywhere in the world.antique996, @tiankonguse, @enginefuture, @finwo, @tarasyarema, @lawrencedark, @vashio, @dchem, @inso-, @mardab, @arhantsundaram, @wangke0809, @vimsuner, @cjqhenry14, @hqm1988, @SinSiXX, @felipek, @davidhartleyllc, @Rob-Rychs, @daviddahl

**Comment by kasparsd** [comment]
Reported the above ☝️ as spam.
